docs/data/tree-view/simple-tree-view/customization/CustomizedTreeView.tsx

```diff
@@ -130,6 +134,9 @@ const StyledTreeItem = React.forwardRef(function StyledTreeItem(
 
   return (
     <StyledTreeItemRoot
+      slots={{
```

Comment:
One could argue that using `useTreeItem` and re-building the UI is better here instead of passing everything inside the label, but it would complexity the demo so I'm in favor of keeping it like this